AcademyResidentInformationApi/V1/Infrastructure/Occupation.cs

```diff
@@ -0,0 +1,17 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace AcademyResidentInformationApi.V1.Infrastructure
+{
+    [Table("ctoccupation", Schema = "dbo")]
```

Comment:
Why are we using the word "occupation" for this element?

Reply:
This is the name of the table in Academy